save-cli/src/commonMain/kotlin/org/cqfn/save/cli/Config.kt

```diff
@@ -0,0 +1,53 @@
+/**
+ * Utilities to work with SAVE config in CLI mode
+ */
+
+package org.cqfn.save.cli
+
+import org.cqfn.save.core.config.ReportType
+import org.cqfn.save.core.config.SaveConfig
+
+import okio.Path.Companion.toPath
+
+import kotlinx.cli.ArgParser
+import kotlinx.cli.ArgType
+import kotlinx.cli.default
+
+/**
+ * @param args CLI args
+ * @return an instance of [SaveConfig]
+ */
+fun createConfigFromArgs(args: Array<String>): SaveConfig {
+    val parser = ArgParser("save")
+
+    val configPath by parser.option(
```

Comment:
just "config", cli should not have complex argument names. Also camel case is not applicable for cli